md/installation_guide.md

```diff
@@ -94,6 +94,16 @@ openssl pkcs12 -export -in /etc/letsencrypt/live/{domain}/fullchain.pem -inkey /
 
 Then when using the installer provide this generated file as a keystore, the same password for the keystore and the key, and the same alias.
 
+#### Renew certificate
+
+When the certificate expires, it can be renewed by reinstalling the backend (not the storage).
```

Comment:
renewing needs to be done on LetsEncrypt (for example), right? It looks like Bonita can renew the certificate.

Reply:
ok, i'll change that